src/js/angular/repositories/templates/ontop-repo.html

```diff
@@ -64,21 +66,21 @@ <h4>Ontop configuration</h4>
                    placeholder="{{isRequiredOntopRepoFile(file) ? 'This file is required' : ''}}"
                    ng-model="ontopRepoFileNames[file]">
         </div>
-        <div class="form-group col-xs-5 col-sm-4 col-md-2 col-md-offset-3">
+        <div class="col-xs-4 col-md-2">
             <div class="btn btn-outline-primary btn-sm">
                 <div ngf-select ngf-change="uploadOntopRepoFile($files, file)" ngf-multiple="false "
                      class="selectFiles pointer clearfix">
                     <em ng-hide="uploadFileLoader" class="fa fa-upload fa-2x"></em>&nbsp;
                     <div ng-show="uploadFileLoader" onto-loader size="24" class="spinner-container ot-loader"></div>
-                    <span class="pageInfo" style="vertical-align: super; color: #8a8a8a;">Upload file</span>
+                    <span class="pageInfo" style="vertical-align: super; color: #8a8a8a;">Upload file...</span>
                 </div>
             </div>
         </div>
     </div>
     <div>
-        <div class="form-group row indented-div" ng-if="editRepoPage">
-            <div class="checkbox offset-xs-3 offset-md-2">
-                <label class="col-xs-4 col-sm-4 col-md-4 col-md-offset-3">
+        <div class="form-group row" ng-if="editRepoPage">
+            <div class="checkbox col-xs-12">
+                <label tooltip="Restarts the repository to apply the changes.">
```

Comment:
What about changing the tooltip to `Check to restart repository when applying the changes` as this is not a button that restarts the repository.

Reply:
Changed the message to a similar one to what you suggested.